### PR TITLE
CPP2-5 UART communication

### PR DESCRIPTION
**Pull request by TheH3ROo**

### CODE REVIEW

File: NUCLEO-144_app/Inc/stm32f7xx_hal_conf.h [hunk 1]
```diff
@@ -190,8 +190,8 @@
 
 /* Section 2: PHY configuration section */
 
-/* LAN8742A_PHY_ADDRESS Address*/ 
-#define LAN8742A_PHY_ADDRESS           0
+/* DP83848_PHY_ADDRESS Address*/ 
+#define DP83848_PHY_ADDRESS           0x01U
```
Comment (johnyka) [inline NUCLEO-144_app/Inc/stm32f7xx_hal_conf.h]: what does this change mean?

File: NUCLEO-144_app/Inc/stm32f7xx_hal_conf.h [hunk 2]
```diff
@@ -202,8 +202,8 @@
 
 /* Section 3: Common PHY Registers */
 
-#define PHY_BCR                         ((uint16_t)0x00U)    /*!< Transceiver Basic Control Register   */
-#define PHY_BSR                         ((uint16_t)0x01U)    /*!< Transceiver Basic Status Register    */
+#define PHY_BCR                         ((uint16_t)0x0000U)    /*!< Transceiver Basic Control Register   */
+#define PHY_BSR                         ((uint16_t)0x0001U)    /*!< Transceiver Basic Status Register    */
```
Comment (johnyka) [inline NUCLEO-144_app/Inc/stm32f7xx_hal_conf.h]: why do we need to change this?

File: NUCLEO-144_app/SDC-NUCLEO Run.cfg [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# This is an NUCLEO-F746ZG board with a single STM32F746ZGTx chip
```
Comment (johnyka) [inline NUCLEO-144_app/SDC-NUCLEO Run.cfg]: can we put this file in gitignore?

File: NUCLEO-144_app/SDC-NUCLEO.ioc [hunk 4]
```diff
@@ -29,30 +24,29 @@ KeepUserPlacement=true
 Mcu.Family=STM32F7
 Mcu.IP0=ADC1
 Mcu.IP1=ADC2
-Mcu.IP10=SYS
```
Comment (johnyka) [inline NUCLEO-144_app/SDC-NUCLEO.ioc]: did you try that tasks still work without this?

Comment (TheH3ROo) [reply NUCLEO-144_app/SDC-NUCLEO.ioc]: It still works my friend.

File: NUCLEO-144_app/SDC-NUCLEO Run.cfg (hunk 3 above)
Comment (TheH3ROo) [reply NUCLEO-144_app/SDC-NUCLEO Run.cfg]: I just did.

**Pull request merged**